routes/faucet.ts

```diff
@@ -23,12 +28,34 @@ const counterDripError = new client.Counter({
 router.post(
   "/",
   ensureAuthenticated,
+  decorateGithubUser,
   blockedAddresses,
   rateLimit,
   async (req: any, res: any, next: any) => {
     const { address } = req.body;
 
     try {
+      if (req.user.github) {
+        const oneQuarter = 7.776e9;
+        const dateSince =
+          new Date().getTime() - new Date(req.user.github.created_at).getTime();
```

Comment:
Worth bounds checking `.created_at` to make sure it isn't 0 or something like that?

Reply:
No need since using `Date` constructor. Invalid construction string, will result in `Invalid Date`, and `getTime` will return `NaN`, valid date - `NaN` results in `NaN`. `NaN` is never greater than, less than, or equal to, but is otherwise treated as a Numeric value. This code path results in proper checking, and resolution through the comparative statement.

Reply:
0 correlates to ~1970 or so.
```
new Date(0)
Wed Dec 31 1969 18:00:00 GMT-0600 (Central Standard Time)
```

A github account can't be from 1970.

Reply:
This value is a UTC string. I can check if time separator is missing or something similar "DD/MM/YYYYTHH:MM" or something some check, it shouldn't ever be a single numerical value.

Reply:
checking if string is ascii format before passing to `Date()`